resources/user-agents/browsers/mozilla-browser/mozilla-browser-0-6-1-8.json

```diff
@@ -0,0 +1,85 @@
+{
+  "division": "Firefox #MAJORVER#.#MINORVER#",
+  "versions": ["1.8", "1.7", "1.6", "1.5", "1.4", "1.3", "1.2", "1.1", "1.0", "0.9", "0.8", "0.7", "0.6"],
```

Comment:
Part of the "being very specific" I mention in the PR's description are the versions here.  I wanted to make sure that only versions of the Mozilla Browser that were released were listed here. This list comes from here: https://en.wikipedia.org/wiki/History_of_Mozilla_Application_Suite#Release_history

1.8 was technically not released in a "stable" form, but the alpha builds do appear to have been made available for download, so I put that one in.  I did not put 1.9 in since it seems like that was probably never released.